src/main/java/edu/cornell/kfs/fp/document/validation/impl/CuDisbursementVoucherNonresidentInformationValidation.java

```diff
@@ -281,7 +282,7 @@ public boolean validate(AttributedDocumentEvent event) {
                 if (!(nonResidentAlienTax.getStateIncomeTaxPercent().equals(KualiDecimal.ZERO))
                         || !(nonResidentAlienTax.getFederalIncomeTaxPercent().equals(KualiDecimal.ZERO))) {
```

Comment:
Another spot that needs to use BigDecimal instead of KualiDecimal. This section also needs to be fixed in base code.